### ISSUE TITLE
loopSound property doesn't work (probably wrong data type)

### ISSUE THREAD

**Issue by KangaRoo1372** [openliero/openliero#41]
Hey guys,

I made some tests with the loopSound parameter and it seems that it does not work correctly (actually it does not work at all).
In original Liero, loopAnim meant that sound of the weapon would loop when the fire key was pressed, and immediately stop when the key was released. Meanwhile, in openliero (similarly to Liero 1.36) this effect practically does not work at all.

I suspect that the reason for this is the incorrect specification of the data type of this parameter as int, instead of boolean. My suspicion stems from the fact that:

1) in the weapon.hpp and common.model.hpp files the launchSound parameter is specified as int (in my opinion this is wrong), while in the common_exereader.cpp file it is already specified as boolean (in my opinion this is correct);
2) in many editors for the classic Liero (e.g. LieroKit) the loopAnim parameter was treated as a boolean type.

**Comment by grawlinson** [comment]
You’re correct in your reasoning, thanks for confirming what I originally thought about that particular parameter!

**Comment by grawlinson** [comment]
Just had a look at the sfx code and I noped straight outta there. :cry:

**Comment by TimmermanV** [comment]
I changed this setting locally from int to bool while working on my other change (changing serialization of sounds from index to sound name/id). I'll see if that makes any difference. I expect this is not the change that will fix it though.
